src/Scores.h

```diff
@@ -66,9 +66,19 @@ class ScoreHolder {
   inline bool isTarget() const { return label != -1; }
   inline bool isDecoy() const { return label == -1; }
   void printPSM(ostream& os, bool printDecoys, bool printExpMass);
+  void printPepXML(ostream& os, map<char,float> &aaWeight, int index);
   void printPeptide(ostream& os, bool printDecoys, bool printExpMass, Scores& fullset);
+  std::string getCharge(std::string id);
+};
+struct less_than_base_name
```

Comment:
Not sure what our convention is for struct names, I think it was also camelCase

Reply:
Fixed